src/test/java/org/cactoos/io/WriterToTest.java

```diff
@@ -51,22 +50,20 @@
     public void writesLargeContentToFile() throws IOException {
         final Path temp = this.folder.newFile("cactoos-1.txt-1")
             .toPath();
-        MatcherAssert.assertThat(
+        new Assertion<>(
             "Can't copy Input to Output and return Input",
-            new TextOf(
-                new TeeInput(
-                    new ResourceOf("org/cactoos/large-text.txt"),
-                    new WriterAsOutput(new WriterTo(temp))
+            () -> new TextOf(
+                new Sticky(
```

Comment:
@Iprogrammerr Is necessary use the Sticky decorator here?

Reply:
@fabriciofx Same as above.